### DISCUSSION TITLE
Which config.h for my Daikin Altherma 3R ECH2O 508 (Model EHSX08P50EF)?

### DISCUSSION THREAD

**Discussion by Schmiddy8888** [raomin/ESPAltherma#250] [Q&A]
Hi all,

successfully installed ESPAltherma and I'm really happy with it. Unfortunately some values are missing/0 so I'm thinking, my choice of the config #include "def/ALTHERMA(HPSU6_ULTRA).h is not the right one.

Could anyone please be so kind and tell me, which configuration is the right one for my Model EHSX08P50EF (Daikin Altherma 3R ECH2O 508 6kW)?

Thanks in advance.

Maybe it's useful to create a table collecting all Daikin Modelnumbers and assign it to the config.h-Files. I've recognized a lot of questions concerning missing values/fields.

Best regards from germany
Lars

**Comment by KalaNagHTD** [comment]
You have the recent version of the ECH2O/HPSU. The #include you have chosen is the best!Am 18.02.2023 um 23:36 schrieb Schmiddy8888 ***@***.***>:﻿
Hi all,
successfully installed ESPAltherma and I'm really happy with it. Unfortunately some values are missing/0 so I'm thinking, my choice of the config #include "def/ALTHERMA(HPSU6_ULTRA).h is not the right one.
Could anyone please be so kind and tell me, which configuration is the right one for my Model EHSX08P50EF (Daikin Altherma 3R ECH2O 508 6kW)?
Thanks in advance.
Maybe it's useful to create a table collecting all Daikin Modelnumbers and assign it to the config.h-Files. I've recognized a lot of questions concerning missing values/fields.
Best regards from germany
Lars

**Reply by blumpp** [reply]
Hi.

I am also struggeling with the right config.h file for this machine (EHSX08P50EF).
The file mentioned above is not available anymore in the latest release, right?
Does anybody know which header to use or how to get valid data?

Thanks!

Bjoern
